tasks/run_agent.rb

```diff
@@ -0,0 +1,160 @@
+#!/opt/puppetlabs/puppet/bin/ruby
+
+require 'open3'
+require 'etc'
+require 'json'
+require 'yaml'
+require_relative '../../ruby_task_helper/files/task_helper.rb'
```

Comment:
Should I add [`puppetlabs/ruby_task_helper`](https://forge.puppet.com/puppetlabs/ruby_task_helper) to the metadata.json?
As far as the module is concerned, it's a 'soft' dependency.  But I also don't envisage a module namespace clash, so adding it to the metadata might be more helpful to users.